DEFAULT_CONFIG.json5

```diff
@@ -25,6 +25,24 @@
     endpoints: [
       // "<proto>/<address>"
     ],
+    //  /// Global connect retry configuration, 
+    //  /// The retry configuration can also be specified for the separate endpoint ("retry_" + field name)
+    //  /// it will override the global one
+    //  /// E.g. tcp/192.168.0.1:7447#retry_try_timeout_ms=20000;retry_period_max_ms=10000"
+    //  retry: { 
+    //    /// global timeout for full connect cycle (currently not supported)
+    //    timeout_ms: 10000,
+    //    /// timeout for one connect try
+    //    try_timeout_ms: 10000,
+    //    /// intial wait timeout until next connect try
+    //    period_init_ms: 1000,
+    //    /// maximum wait timeout until next connect try
+    //    period_max_ms: 4000,
+    //    /// increase factor for the next timeout until nexti connect try
+    //    period_increase_factor: 2,
+    //    /// if global timeout exceed, exit from application (currently not supported)
+    //    fail_exit: false,
```

Comment:
fail_exit have different defaults for clients, peers and routers. This should be reflected here.